### ISSUE TITLE
[enhancement] Prompt for Rosetta on fresh M1 macs

### ISSUE THREAD

**Issue by coolaj86** [webinstall/webi-installers#609]
When we fallback to x64 on Apple M1 we should check if rosetta is installed / prompt to install it:

```sh
if ! test -d /usr/libexec/rosetta; then
    prompt "Install Rosetta 2? (ctrl+c to cancel, any key to continue)"
    softwareupdate --install-rosetta --agree-to-license
fi
```

See also https://apple.stackexchange.com/a/427996

**Comment by coolaj86** [comment]
There's a probably a way to have it run only for fallback arch.

**Comment by coolaj86** [comment]
This is available as part of #768, but not on-by-default. We'll wait for #756 for that.

Closing here and leaving as a checkmark in #756.
